### PR TITLE
Use kebab case for project names in user manual

### PR DESCRIPTION
**Pull request by donat**
Most of the project names are defined in the samples. The naming conversion happens in #14198. This PR changes the user guide only.

### CODE REVIEW

File: subprojects/docs/src/docs/userguide/authoring-builds/multi_project_builds.adoc [hunk 1]
```diff
@@ -391,7 +391,7 @@ include::{snippetsPath}/multiproject/partialTasks/tests/multiprojectPartialTasks
 ----
 ====
 
-The build is executed from the `water` project. Neither `water` nor `tropicalFish` have a task with the name `distanceToIceberg`. Gradle does not care. The simple rule mentioned already above is: Execute all tasks down the hierarchy which have this name. Only complain if there is _no_ such task!
+The build is executed from the `water` project. Neither `water` nor `tropicalfish` have a task with the name `distanceToIceberg`. Gradle does not care. The simple rule mentioned already above is: Execute all tasks down the hierarchy which have this name. Only complain if there is _no_ such task!
```
Comment (rieske) [inline subprojects/docs/src/docs/userguide/authoring-builds/multi_project_builds.adoc:394]: Why no kebab for this one?

Comment (donat) [reply subprojects/docs/src/docs/userguide/authoring-builds/multi_project_builds.adoc:394]: all lowercase is kebab :) 
With all seriousness, I've done this for consistency: there was already a project called `bluewhale` (and not `blueWhale`).

Comment (jjohannes) [reply subprojects/docs/src/docs/userguide/authoring-builds/multi_project_builds.adoc:394]: I think all fish and water references should really be remove in favour of samples based on real use cases. So I think this really does not matter as we will hopefully update this part of the docs soon.

**Review by jjohannes — APPROVED** [review]

**Comment by donat** [comment]
@bot-gradle test this

**Pull request merged**